.gitignore

```diff
@@ -11,6 +11,8 @@ build/
 *.log.*
 snowshu_barf_output/
 keyfile.json
+credentials.yml
+replica.yml
```

Comment:
We do still have two files named `replica.yml` and `credentials.yml` in the `templates` directory, so we should probably not ignore them for the time being.

Reply:
Update the file